CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing Guide
+
+This is part of the [Porter][porter] project. If you are a new contributor,
+check out our [New Contributor Guide][new-contrib]. The Porter [Contributing
+Guide][contrib] also has lots of information about how to interact with the
+project.
+
+[porter]: https://github.com/deislabs/porter
+[new-contrib]: https://porter.sh/contribute
+[contrib]: https://github.com/deislabs/porter/blob/master/CONTRIBUTING.md
+
+---
+
+* [Initial setup](#initial-setup)
+* [Makefile explained](#makefile-explained)
+
+---
+
+# Initial setup
+
+You need to have [porter installed](https://porter.sh/install) first. Then run
+`make build install`. This will build and install the mixin into your porter
```

Comment:
`This will build and install the mixin...` <- is this intended?  Or do we intend to say it will install the porter client & runtime and all default mixins?

Reply:
This is describing the makefile for just the helm mixin, not the porter makefile. 

So in this case, I think the description is accurate. It builds the mixins client and runtime, which I'm simplifying a bit as building your mixin, and then it installs it into the porter home directory.